### ISSUE TITLE
We lost the ability to export selected products from product-list

### ISSUE THREAD

**Issue by exeqtwasnotfree** [atrocore/atrocore#1066]
After selection of a couple of products, before we was able to select the action-drop-down, and select to export.
This is now gone since at least v1.8.12
Where did it go? :-)

**Comment by atrolex** [comment]
Hi, we added a new abstaction level ("configurable buttons").
Now we can configure a lot of actions by youself:
https://demo.atropim.com/#Action/create

**Comment by exeqtwasnotfree** [comment]
Okey, I got it working and it seems better.
I wonder, have i missed somewhere to read about what changes you do?
It´s a bit hard to have a LAB-server and guess what have changed.

**Comment by JoacoALV** [comment]
I agree with @exeqtwasnotfree, they make changes, and we cannot keep track of what they consist of or whether they will affect our processes. How can we stay informed?

Thank you in advance.

**Comment by marcus-at-localhost** [comment]
> I agree with @exeqtwasnotfree, they make changes, and we cannot keep track of what they consist of or whether they will affect our processes. How can we stay informed?
> 
> Thank you in advance.

There is a RSS Feeds for changes in their docs: https://help.atrocore.com/#rrs-feeds. Personally, I turned off auto-updates and get all notifications of changes here via github. Most of the time I have no clue what changed because I'm not familiar with the code or their roadmap and I'm really anxious to press the update button after 3 months of not being able to keep up. But at least I know something is changing every day.

On the other side, I understand them to move fast (and break things).

**Comment by JoacoALV** [comment]
@marcus-at-localhost  thank you very much for your answer. It is useful.

**Comment by AnnZast** [comment]
Hello @exeqtwasnotfree, we decided to add a system action that will come out of the box with the Export Feeds module to export selected records. So that users do not lose the functions they are used to. Thank you for your comments.

**Comment by AnnZast** [comment]
We have returned the ability to export selected records. Now this Action is available by default.
